assets/css/bpmn-js-bpmnlint.css

```diff
@@ -148,12 +148,18 @@
   --icon-color: var(--bjsl-color-info);
 }
 
-.bjsl-issues a {
+.bjsl-issues .message {
   color: var(--bjsl-font-color);
-  margin-left: 8px;
+  margin-left: .5em;

```

Comment:
Why should we tie the margin to the font size?

Reply:
To ensure it is proportional to the font size: 

![capture rCnwXL_optimized](https://github.com/user-attachments/assets/7f8982b0-f403-4660-b287-51bcef540096)

Reply:
Yeah, I've just tested this and indeed it looks great
![image](https://github.com/user-attachments/assets/a0aa0aaf-71bd-4e61-8b82-8cebeed38d7d)